mypyc/test-data/fixtures/ir.py

```diff
@@ -289,6 +289,12 @@ def zip(x: Iterable[T], y: Iterable[S], z: Iterable[V]) -> Iterator[Tuple[T, S,
 def eval(e: str) -> Any: ...
 def abs(x: float) -> float: ...
 def exit() -> None: ...
+@overload
+def min(x: int, y: int) -> int: ...
+@overload
+def min(x: float, y: float) -> float: ...
+@overload
+def min(x: str, y: str) -> str: ...
```

Comment:
You can use `T` (already defined in this file) to make them simple.

Reply:
Added